templates/mfa/index.html

```diff
@@ -0,0 +1,69 @@
+{% extends "mfa/base_manage.html" %}
+
+{# https://github.com/pennersr/django-allauth/blob/main/allauth/templates/mfa/index.html #}
+
+
+{% load allauth %}
+{% load i18n %}
+
+
+{% block content %}
+
+  <h1>{% block heading %}{% trans "MFA" %}{% endblock heading %}</h1>
+
+  <p>{% trans 'Configure or update your multi-factor authentication settings.' %}</p>
+
+
+  {% if "totp" in MFA_SUPPORTED_TYPES %}
+
+  <section class="my-5" id="mfa-totp">
+    <h3>{% trans "Authenticator App" %}</h3>
+
+    {% url 'mfa_deactivate_totp' as deactivate_url %}
+    {% url 'mfa_activate_totp' as activate_url %}
+
+    {% if authenticators.totp %}
+      <p>
+        {% blocktrans trimmed %}
+          Two-factor authentication using an authenticator app is enabled.
+        {% endblocktrans %}
+      </p>
+      <p class="mb-0"><a href="{{ deactivate_url }}" class="btn btn-sm btn-danger">{% trans 'Deactivate MFA' %}</a></p>
```

Comment:
There's a bunch of places where we're using this acronym, but seems ambiguous for users, and never defined in the UI?